__tests__/LoginAvatar-test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {LoginAvatar} from '../src/LoginAvatar';
+import { mountWithIntl, shallowWithIntl } from '../src/helpers/intl-enzyme-test-helper.js';
+
+function setup() {
+  const props = {
+    isAuthenticated: true,
+    dispatch: jest.fn(),
+    intl: {
+        formatDate: jest.fn(),
+        formatTime: jest.fn(),
+        formatRelative: jest.fn(),
+        formatNumber: jest.fn(),
+        formatPlural: jest.fn(),
+        formatMessage: jest.fn(),
+        formatHTMLMessage: jest.fn(),
+        now: jest.fn(),
+    },
+    profile: {}
+  }
+
+  const enzymeWrapper = shallowWithIntl(<LoginAvatar {...props} />)
+
+  return {
+    props,
+    enzymeWrapper
+  }
+}
+
+describe('components', () => {
+  describe('LoginAvatar', () => {
+    it('should render div.avatar if logged in', () => {
```

Comment:
the 'if logged in' bit makes me think that this test either will only run when logged in, which isn't obvious from this file.